x/validators/handler.go

```diff
@@ -81,7 +81,7 @@ func (h updateHandler) validate(ctx weave.Context, store weave.KVStore, tx weave
 		return nil, resUpdates, errors.Wrap(errors.ErrUnauthorized, "no permission")
 	}
 
-	updates, err := GetValidatorUpdates(store)
+	updates, err := weave.GetValidatorUpdates(store)
```

Comment:
Cannot comment on the line I want (9 below)...

But if we add `ValidatorUpdates.Get(PubKey) *ValidatorUpdate`, this becomes easy:

```golang
for _, v := range diff {
  match := updates.Get(v.PubKey)
  if !match && v.Power == 0 {
    return nil, nil, errors.Wrap(errors.ErrInput, "setting unknown validator power to 0")
  }
  if match && v.Power == match.Power {
    return nil, nil, errors.Wrap(errors.ErrInput, "same validator power")
  }
}
updates.ValidatorUpdates = append(updates.ValidatorUpdates, diff...)
updates = updates.Deduplicate(true)
```

At least this seems cleaner to me